IPython/core/debugger.py

```diff
@@ -40,6 +41,9 @@
 from IPython.testing.skipdoctest import skip_doctest
 
 
+RGX_EXTRA_INDENT = re.compile('(?<=\n)\s+')
```

Comment:
Style point - let's put this near the function that uses it, so it's easier to see them both together.